client/src/modules/search-results/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card, Grid, Typography } from '@material-ui/core';
+import Layout from '../home/shared/components/Layout';
+
+const SearchResults: React.FC = () => {
+  // TODO: Replace with `useEventsQuery()`!
+  const data = {
+    searchResults: [],
+    searchTerm: 'London',
+  };
+
+  return (
+    <Layout>
+      <h1>
+        Search results for <q>{data?.searchTerm}</q>
+      </h1>
+      <Grid container spacing={2}>
+        <Grid item xs={10}>
+          {data?.searchResults.length === 0 && (
+            <Typography gutterButtom variant="p" component="p">
+              Nothing found.
+            </Typography>
+          )}
+          {data?.searchResults.map(searchResult => (
+            <Card>
```

Comment:
We should have a `key` prop on this component

Reply:
Do you have suggestions on its value? Right now, there aren't any real search results.
I'd need to wire it up with GraphQL (and then could look into adding a `key` prop).

Reply:
Ah, just realized the result array is empty. I'd add a to-do here then to make sure it doesn't get left behind.